packages/wallet/src/TxBuilder/types.ts

```diff
@@ -173,12 +188,18 @@ export interface TxBuilder {
    * In case validations fail, it creates a `TxBodyValidationError` instead of `ValidTxBody`.
    *
    * @returns {Promise<MaybeValidTx>}
-   * - In case it is a `ValidTx`, it can be used to sign and submit the transaction. This is a snapshot of transaction.
-   *   Further changes done via TxBuilder, will not update this snapshot.
-   * - In case of it is an `InvalidTx`, it embeds a TxBodyValidationError with more details
+   * - In case it is a {@link ValidTx}, it can be used to sign and submit the transaction.
+   *   This is a snapshot of transaction. Further changes done via TxBuilder, will not update this snapshot.
+   *   Once a transaction is sucessfully submitted, the TxBuilder cannot be reused, to avoid.
+   * - In case of it is an `InvalidTx`, it embeds a TxBodyValidationError with more details.
+   * - `InvalidTx` embeds a {@link TxBuilderDisabled} error if a transaction built by this TxBuilder
+   *   was already successfully submitted.
    */
   build(): Promise<MaybeValidTx>;
 
+  /** Force enable TxBuilder to be reused, even if a transaction with this configuration was successfully submitted. */
+  forceEnable(): TxBuilder;
```

Comment:
Let's try to find a better term for this. 1 suggestion below, but not sure about this. Thoughts @rhyslbw ?
```suggestion
  isDestroyed: boolean;
  restore(): TxBuilder;
```

Reply:
Suggestion here:
https://github.com/input-output-hk/cardano-js-sdk/pull/434#discussion_r969158433

Reply:
It would be nice to have a getter, but `isDestroyed` is not very accurate. It's still there, you can use most of its methods, only build() is restricted.